assets/prefabs/items/assemblyProcesses/AssembleWoodenShoe.prefab

```diff
@@ -0,0 +1,19 @@
+{
+  "Parent": "CrudeAssemblingBaseProcessPart",
+  "ProcessDefinition": {
+    "processType": "ManualLabor:CrudeAssembling"
+  },
+  "RequirementInput": {
+      "requirements": ["CrudeSaw"]
+    },
+  "InventoryInput": {
+    "itemCounts": {
+      "ManualLabor:Plank": 4
```

Comment:
I'd think you'd maybe need some kind of (leather?) straps to fasten the shoe to the foot :thinking:

Reply:
Strings should work I guess... like just tie them to the foot..